Tools/scripts/pathfix.py

```diff
@@ -164,12 +180,28 @@ def fix(filename):
     # Return success
     return 0
 
+
+def parse_shebang(shebangline):
+    end = len(shebangline)
+    start = shebangline.find(b' -', 0, end)  # .find() returns index at space
+    if start == -1:
+        return b'', b''
+    print(shebangline[(start + 2):end])
+    return shebangline[(start + 2):end]
+
+
 def fixline(line):
+    global keep_flags
     if not line.startswith(b'#!'):
         return line
+
     if b"python" not in line:
         return line
+    if keep_flags:
+        flags = parse_shebang(line)
+        return b'#! ' + new_interpreter + b" -" + flags
     return b'#! ' + new_interpreter + b'\n'
```

Comment:
I would prefer to have:

```
    if keep_flags:
        flags = parse_shebang(line)
    else:
        flags = b''
    return b'#! ' + new_interpreter + flags + b'\n'
```

So parse_shebang() must remove the trailing newline.

Reply:
Ping @PatrikKopkan?

Reply:
I have zero idea how I deleted. In editor I still have:
        ```return b'#! ' + new_interpreter + b' -' + flags + b'\n'```